tutorials/tts/Audio_Codec_Training.ipynb

```diff
@@ -130,26 +133,29 @@
         "*   **audio_codec_*.yaml**: Audio codec configurations optimized for various sampling rates.\n",
         "*   **mel_codec_*.yaml**: A mel-spectrogram based codec designed to maximize the performance of speech synthesis models.\n",
         "*   **encodec_*.yaml**: A reproduction of the original [EnCodec](https://arxiv.org/abs/2210.13438) model setup.\n",
+        "*   **audio_codec_low_frame-rate_22050.yaml**: The Low Frame-rate Speech Codec: an audio codec that achieves high-quality audio compression with a 1.89 kbps bitrate and 21.5 frames per second.\n",
```

Comment:
Nitpick: I would favor underscore over hyphen in a file name, like "frame_rate".